scripts/clean-db-dump.sql

```diff
@@ -144,27 +144,22 @@ SELECT
         '"respondToEmailAddress": "example-email@gov.uk"' ||
       '}'
     )::JSON,
-    eligible_brief_supplier_pairings.brief_id,
+        eligible_brief_supplier_pairings.supplier_id,
```

Comment:
This doesn't seem like a flake8 change? It looks over-indented, it changes a key, and it duplicates the key below?

Reply:
Are any of the changes in this file intentional?

Reply:
Damn, they snuck in there, good spot.

Reply:
Done 👍